.travis.yml

```diff
@@ -29,15 +29,41 @@ env:
     - SEQ_PG_PW=sequelize_test
     - SEQ_PG_HOST=127.0.0.1
     - SEQ_PG_PORT=8998
+    # oracle info
+    - NODE_ORACLEDB_HOST: localhost
+
+before_install:
+  - |
+    if [ $ORACLE_VER ]; then 
+      mkdir -p /opt/oracle && 
+      cd /opt/oracle && 
+      wget https://s3.amazonaws.com/sequelize/instantclient-basic-linux.x64-12.1.0.2.0.zip &&
+      wget https://s3.amazonaws.com/sequelize/instantclient-sdk-linux.x64-12.1.0.2.0.zip &&
```

Comment:
who owns this `https://s3.amazonaws.com/sequelize/` ? Dont we have any official channel ?

Reply:
I'm not sure who owns it, but I _do_ know that Oracle doesn't let you download any of their Instant Client utilities. The only semi-trustless ways to do this, as far as I understand:
- Add the above files to the repository (gross!).
- Use a torrent or some other content-addressable store.

Reply:
http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html ?

Reply:
nvm, this needs login. But I know for sure I cant let this package merged in current state. May be we can manage this an one of repo, under sequelize ? I will look into it.

Reply:
Sorry, I should've been more clear -- Oracle doesn't let you download any of their Instant Client utilities non-interactively (i.e. with `wget` or `curl`). Clicking those links forces an interactive license agreement and redirects you to a login page. There used to be ways around it, but I wasn't able to find a workable solution to this problem without just storing the downloads in Git.

Reply:
After the PR was created, @janmeier  created (thanks to him) the build for Oracle. So I assume he has the right to update the package.
@christianbundy The link is not an Oracle link, it's aws; this means that the file has been downloaded once and put on another server where we have access.

Reply:
I own that bucket :)

There is https://github.com/cbandy/travis-oracle, but as I remember, it was pretty slow and would not always work when i tested it (it uses a browser to log in and accept the terms)